NuRadioReco/test/test_examples.sh

```diff
@@ -15,22 +15,22 @@ rm -rf *deg_12/
 rm output_file.hdf5
 rm output_snr.json
 # Envelope phased array
-cd ../../EnvelopePhasedArray/Effective_volume/
-python3 T01generate_event_list.py minimal
-python3 T02RunPhasedRNO_shortband.py --inputfilename minimal_eventlist.hdf5
-rm minimal_eventlist.hdf5
-rm output*hdf5
-cd ../Noise_trigger_rate
-python3 noise_trigger_rate.py --ntries 10
-# Alias phased array
-cd ../../AliasPhasedArray/SNR_study
-python3 T01_generate_events_simple.py --n_events 10
-python T02SNRNyquist.py input_alias_SNR.hdf5 phased_array_100m_0.25GHz.json config.yaml out.hdf5 out.json --nyquist_zone 2 --upsampling_factor 4 --noise_rate 1
-rm input_alias_SNR.hdf5
-rm out.hdf5
-rm out.json
-cd ../Noise_trigger_rate
-python3 noise_trigger_rate.py --ntries 10
+#cd ../../EnvelopePhasedArray/Effective_volume/
```

Comment:
I think you forgot to undo something here :-)

Reply:
no they don't work anymore due to the phased array refactoring. This needs to be fixed later in the refractor-pa branch